src/DotVVM.Framework/Resources/Scripts/binding-handlers/introduce-alias.ts

```diff
@@ -21,6 +21,11 @@ function createWrapperComputed<T>(accessor: () => KnockoutObservable<T> | T, pro
         }
     });
     computed["wrappedProperty"] = accessor;
+    Object.defineProperty(computed, "state", {
+        get: () => (accessor() as any || {})["state"],
+        configurable: false,
+        enumerable: false
+    })
```

Comment:
We should probably also clone the other properties like `setState`.

Reply:
I will create a new PR that will add other props.